float32 x_bola
float32 y_bola
float32 z_bola
int16 x_pixel
int16 y_pixel
int16 radius
